interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+
+    # We need to allow access to the fuse device here as we're in a child
+    # profile and fuse-support wouldn't help at this point.
+    /dev/fuse rw,
+
+    # container start
+    owner @{PROC}/[0-9]*/fd/ r,
+    owner @{PROC}/[0-9]*/stat r,
+    owner @{PROC}/[0-9]*/ns/ r,
+    /dev/ptmx rw,
+    /dev/pts/[0-9]* rw,
+    capability net_admin,
+    /sys/devices/** r,
+    capability sys_ptrace,
+    / r,
+    capability sys_module, # why?
+    capability kill,
+
+    # Allow liblxc to change to the container child profile
+    # FIXME: Can we get the parent profile name somehow?
+    change_profile unsafe /** -> snap.anbox.container-manager//container,
+
+    mount options=(rw,bind),
+    mount options=(rw,rbind),
+    mount options=(rw,make-slave) -> **,
+    mount options=(rw,make-rslave) -> **,
+    mount options=(rw,make-shared) -> **,
+    mount options=(rw,make-rshared) -> **,
+    mount options=(rw,make-private) -> **,
+    mount options=(rw,make-rprivate) -> **,
+    mount options=(rw,make-unbindable) -> **,
+    mount options=(rw,make-runbindable) -> **,
+}
+
+# The container child profile is used by the actual Android container the Anbox container
+# manager creates and manages through liblxc. The policy here is inheriting from the one
+# LXD defines for its containers. See https://github.com/lxc/lxd/blob/master/lxd/apparmor.go
+profile "container" flags=(attach_disconnected,mediate_deleted) {
+    ### Base profile
+    capability,
+    dbus,
+    file,
+    network,
+    umount,
+
+    # Allow us to receive signals from anywhere.
+    signal (receive),
+
+    # Allow us to send signals to ourselves
+    signal peer=@{profile_name},
+
+    # Allow other processes to read our /proc entries, futexes, perf tracing and
+    # kcmp for now (they will need 'read' in the first place). Administrators can
+    # override with:
+    #   deny ptrace (readby) ...
+    ptrace (readby),
+
+    # Allow other processes to trace us by default (they will need 'trace' in
+    # the first place). Administrators can override with:
+    #   deny ptrace (tracedby) ...
+    ptrace (tracedby),
+
+    # Allow us to ptrace ourselves
+    ptrace peer=@{profile_name},
+
+    # ignore DENIED message on / remount
+    deny mount options=(ro, remount) -> /,
+    deny mount options=(ro, remount, silent) -> /,
+
+    # allow tmpfs mounts everywhere
+    mount fstype=tmpfs,
+
+    # allow hugetlbfs mounts everywhere
+    mount fstype=hugetlbfs,
+
+    # allow mqueue mounts everywhere
+    mount fstype=mqueue,
+
+    # allow fuse mounts everywhere
+    mount fstype=fuse,
+    mount fstype=fuse.*,
+
+    # deny access under /proc/bus to avoid e.g. messing with pci devices directly
+    deny @{PROC}/bus/** wklx,
+
+    # deny writes in /proc/sys/fs but allow binfmt_misc to be mounted
+    mount fstype=binfmt_misc -> /proc/sys/fs/binfmt_misc/,
+    deny @{PROC}/sys/fs/** wklx,
+
+    # allow efivars to be mounted, writing to it will be blocked though
```

Comment:
"... blocked via the above rule and a later rule though"

Though really, it might be nice to move the efivars rule down below to right right here.